app/assets/stylesheets/app/transaction_search.scss

```diff
@@ -10,11 +10,17 @@
 }
 
 .search_form input[type=submit] {
-  clear:both;
-  float:right;
+  clear: both;
+  float: right;
   margin-bottom: 0.5em;
 }
 
+.search_form input[type=submit].submit--float-left {
```

Comment:
@wtfluckey my question here is probably, is there a less awkward way to select the specific element I want (in the html file below), without changing the styling on line 12? Unfortunately the styling on line 12 appears to be re-used enough that it's not worth hunting down all the places that styling is applied to replace it with a more sensible selection scheme.